src/main/java/org/entur/lamassu/model/entities/MultiPolygon.java

```diff
@@ -58,6 +69,27 @@ public static MultiPolygon fromGeoJson(org.geojson.MultiPolygon geometry) {
       )
       .collect(Collectors.toList());
     mapped.setCoordinates(mappedCoordinates);
+
+    var mappedEncodedCoordinates = coordinates
+      .stream()
+      .map(polygon ->
+        polygon
+          .stream()
+          .map(ring ->
+            ring
+              .stream()
+              .map(lngLatAlt ->
+                Point.fromLngLat(lngLatAlt.getLatitude(), lngLatAlt.getLongitude()) // because GeoJson is defined with [lon, lat] instead of [lat, lon] used by PolylineUtils.encode, the order is switched on purpose here
```

Comment:
This is incorrect. The Point-factory method already takes the arguments in the same order as geojson, and flips the coordinates internally. The polyline you have attached below is actually in the wrong place (indian ocean) when it should be in Norway.

Reply:
This is where it should be:

![Screenshot 2024-04-04 at 12 52 18](https://github.com/entur/lamassu/assets/231492/89febed9-5eb8-4389-91f8-4d5919e0e519)

This is where your polyline is:

![Screenshot 2024-04-04 at 12 53 20](https://github.com/entur/lamassu/assets/231492/32b6003c-fd69-4be2-8562-c32b559a48b2)

Reply:
Hmm, yes. I wasn't aware of the Point constructor switch, and see now that in the [Mapbox polyline implementation](https://github.com/mapbox/polyline/blob/master/src/polyline.js), polyline.decode does not flip the order of the coordinates, while polyline.fromGeoJSON does, causing some confusion 😅  Will fix